Updates to Updates (#92)
data-lessons/instructor-training: _episodes/18-preparation.md

In this training, we have discussed many cognitive principles and teaching practices that guide underlie the design and 
implementation of Carpentries Workshops.
We hope you are feeling excited and optimistic about putting 
those concepts to work! 
One thing we have emphasized is that **teaching is a skill** - and a complex one at that. 
Whether you are new to teaching or skilled in certain methods, adding new features takes takes time, effort, and deliberate practice. In this episode, 
we will walk through some suggestions about how to prepare yourself for success in the classroom.

As in other sections, we will not discuss technical preparation. Carefully reviewing the content of your workshop is important! 
However, **it is common for new Instructors to
over-prepare on technical content** -- which can be endless! -- and under-prepare the 
learner-centered elements of their teaching practice. If you encounter questions that you can't answer, 
demonstrating how you Google things may earn surprisingly positive feedback! In contrast, neglecting
to attend to the audience usually has much more serious consequences for learning and morale. When you prepare to teach, 

> we therefore suggest setting aside time *before* deeply reviewing your technical content to **plan your approach to instruction**. 
> 

Reviewer: ```suggestion
we therefore suggest setting aside time *before* deeply reviewing your technical content to **plan your approach to instruction**. 

> ## A note on cutting
>  
> This episode is a common place to trainers to cut while preparing to teach this workshop. 
>  That's not because this is not important, this page is a valuable resource, but we feel this 
>  is one of the sections that trainees can use effectively as a resource when actually preparing
> for a workshop, even without spending a lot of time doing activities on this material in the workshop. 
>
{: .callout}

```

adding acallout as a reminder because I liked @karenword's points during the trainer meeting... very open to rephrasing, but tried to capture the idea.

We hope you are feeling excited and optimistic about putting 
those concepts to work! 
One thing we have emphasized is that **teaching is a skill** - and a complex one at that. 
Whether you are new to teaching or skilled in certain methods, adding new features takes takes time, effort, and deliberate practice. In this episode, 
we will walk through some suggestions about how to prepare yourself for success in the classroom.

As in other sections, we will not discuss technical preparation. Carefully reviewing the content of your workshop is important! 
However, **it is common for new Instructors to
over-prepare on technical content** -- which can be endless! -- and under-prepare the 
learner-centered elements of their teaching practice. If you encounter questions that you can't answer, 
demonstrating how you Google things may earn surprisingly positive feedback! In contrast, neglecting
to attend to the audience usually has much more serious consequences for learning and morale. When you prepare to teach, 
we therefore suggest setting aside time *before* deeply reviewing your technical content to **plan your approach to instruction**. 

> ## A note on cutting
>  

> > This episode is a common place to trainers to cut while preparing to teach this workshop. 
> >  That's not because this is not important, this page is a valuable resource, but we feel this 
> >  is one of the sections that trainees can use effectively as a resource when actually preparing
> > for a workshop, even without spending a lot of time doing activities on this material during 
> > their instructor training workshop. 

Author: ```suggestion
> This episode is a common place to Trainers to cut while preparing to teach. 
>  That's not because this is not important -- this page is a valuable resource -- but we feel this 
>  is one of the sections that trainees can use effectively as a resource when actually preparing
> for a workshop, even without spending a lot of time doing activities on this material during 
> their Instructor Training event. 
```

to attend to the audience usually has much more serious consequences for learning and morale. When you prepare to teach, 
we therefore suggest setting aside time *before* deeply reviewing your technical content to **plan your approach to instruction**. 

> ## A note on cutting
>  

> > This episode is a common place to trainers to cut while preparing to teach this workshop. 
> >  That's not because this is not important, this page is a valuable resource, but we feel this 
> >  is one of the sections that trainees can use effectively as a resource when actually preparing
> > for a workshop, even without spending a lot of time doing activities on this material in the workshop. 
> >

Reviewer: ```suggestion
> This episode is a common place to trainers to cut while preparing to teach this workshop. 
>  That's not because this is not important, this page is a valuable resource, but we feel this 
>  is one of the sections that trainees can use effectively as a resource when actually preparing
> for a workshop, even without spending a lot of time doing activities on this material during 
> their instructor training workshop. 
>
```

make second use of workshop more specific

> Select one learning objective from the episode you've used for teaching practice. Copy it into the etherpad
> then add numbers below your objective to address the following:
>
> 1. Write your learning objective in the Etherpad. 

> > 2. Suppose a learner had mastered this objective, and wanted to do more to advance their mastery of the same topic. Identify an objective they could work towards next.

Author: ```suggestion
> 2. Suppose a learner had mastered this objective, and wanted to do more to advance their mastery of the same topic. Identify an objective they could work towards next. (Note that this should not be a *next step* in a workflow, but rather a more complex objective on the same topic.)
```

> 3. Suppose a learner struggled to meet the specified objective. What might they be missing? Identify one more fundamental thing a learner needs to be able to do in order to be successful in meeting this objective.
>
> This exercise should take about 10 minutes.  
{: .challenge}

## Using Formative Assessments

In your answers above, you may have found yourself using words like "know" and "understand." These are words we are accustomed to using when we talk about 
teaching and learning! However, these terms are so broad that they are actually quite difficult to use when it comes time to *assess* whether a learner has met 
that objective. When do you *know* something? What does it mean to *understand*? 

> When we want to check on the completion of a learning objective, we create tasks, and these tasks usually test knowledge and understanding in very specific ways.

Reviewer 2: tasks that test knowledge and understanding might be useful language to add as a definition in the episode where formative assessment is first mentioned
